src/Lykke.Job.OpsGenie/Settings/JobSettings/OpsGenieSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using Lykke.SettingsReader.Attributes;
+
+namespace Lykke.Job.OpsGenie.Settings.JobSettings
+{
+    public class OpsGenieSettings
+    {
+        public string ApiUrl { get; set; }
```

Comment:
Can we add HttpCheck?

Reply:
This is ops genie api https://api.opsgenie.com/ - i cant find url, where 200 status code returned on GET request